core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -477,7 +477,7 @@ abstract class Chunk[+O] extends Serializable with ChunkPlatform[O] { self =>
 
   override def equals(a: Any): Boolean =
     a match {
-      case c: Chunk[O] =>
+      case c: Chunk[_] =>
```

Comment:
This `O` param was unnecessary here - we don't use it and Dotty correctly complains about it being uncheckable at runtime.